packages/patternfly-4/react-core/src/components/Breadcrumb-2/Breadcrumb.md

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
You can just get rid of everything under Breadcrumb-2 directory.  Looks like it's the old code and the new typescript updates are in Breadcrumb